functions/New-DbaDatabase.ps1

```diff
@@ -0,0 +1,469 @@
+function New-DbaDatabase 
+{
+  <#
+.SYNOPSIS 
+New-DbaADatabase creates a new database
+
+.DESCRIPTION
+New-DbaDatabase creates a new database with a single user filegroup, and the PRIMARY filegroup reserved for system objects.
+It allows creation with multiple files, and sets all growth settings to be fixed size rather than percentage growth.
+
+.PARAMETER SqlInstance
+The SQL server instances to be connected to; can be passed in via the pipeline.
+
+.PARAMETER PsCredential
+Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. To use:
+$scred = Get-Credential, then pass $scred object to the -SqlCredential parameter. 
+To connect as a different Windows user, run PowerShell as that user.
+
+.PARAMETER DatabaseName
+The name of the new database to be created.
+
+.PARAMETER NumberOfFilesInUserFilegroup
+The number of files to create in the user filegroup for the database.  Default is 1.
+
+.PARAMETER UseDefaultFileLocations
+True/False, if true will use the default file locations for data and log files on the SQL Server instance, if false, these are specified
+in the $NonDefaultFileLocation and $NonDefaultLogLocation parameters.
+
+.PARAMETER NonDefaultFileLocation
+The location that data files will be placed if UseDefaultFileLocations is set to false.
+
+.PARAMETER NonDefaultLogLocation
+The location the log file will be placed if UseDefaultFileLocations is set to False.
+ 
+.PARAMETER Collation
```

Comment:
You can get the available collations by connecting to a server, and using the `EnumCollations()` method of the server object. You're right, though, it is a BIG list... the server I happened to have open returns 3887 entries...

```powershell
> $server = Connect-DbaSqlServer sql2014
> $server.EnumCollations().Rows.Count
3887
```